/cm/shared/apps/python/3.5.2/lib/python3.5/site-packages/h5py/__init__.py:34: FutureWarning: Conversion of the second argument of issubdtype from `float` to `np.floating` is deprecated. In future, it will be treated as `np.float64 == np.dtype(float).type`.
  from ._conv import register_converters as _register_converters
parameters: {'seed': 1337, 'freeze_pre_trained_layers': False, 'depth': 9, 'joint_test': 0, 'joint_training': True, 'transfer_weights': False, 'model_folder': 'models/VDCNN/VDCNN_ng20---ag_news_depth@9/10', 'test_only': 0, 'test_batch_size': 128, 'combined_datasets': 'ng20---ag_news', 'maxlen': 1024, 'last_pooling_layer': 'k-max-pooling', 'model_save_path': 'models/VDCNN/VDCNN_ng20---ag_news_depth@9/10', 'num_prev_classes': 20, 'num_embedding_features': 100, 'class_weights': None, 'chunk_size': 2048, 'iterations': 50000, 'test_dataset': 'ng20', 'model_load_path': 'models/VDCNN/ag_news-2000_model.pt', 'lr': 0.01, 'shuffle': True, 'joint_ratio': 0.1, 'dataset': 'ag_news', 'lr_halve_interval': 15000.0, 'transfer_lr': 0.001, 'validation_ratio': 0.1, 'shortcut': False, 'gpu': True, 'batch_size': 128, 'test_interval': 1000}
Joint Training !
dataset: Newsgroup20, n_classes: 20
  - loading dataset...
dataset: AgNews, n_classes: 4
  - loading dataset...
Both datasets loaded, going to mix ...
Transfer Data Size: 120000
Transfer Ratio Size: 12000
Root dataset Test length - sentences 5645, labels 5645
Transfer dataset Test length - sentences 7600, labels 7600
Unused dataset Test length - sentences 108000, labels 108000
Mixed dataset Test length - sentences 121245, labels 121245
Mixed dataset Train length - sentences 25179, labels 25179
 Joint Training: Txt vectorization...
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 282, in joint_train
    logger.info("n_txt_feats before overriding: ", n_txt_feats)
Message: 'n_txt_feats before overriding: '
Arguments: (87,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 282, in joint_train
    logger.info("n_txt_feats before overriding: ", n_txt_feats)
Message: 'n_txt_feats before overriding: '
Arguments: (87,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 284, in joint_train
    logger.info("Number of classes in the mixed dataset are: ", n_classes, type(n_classes))
Message: 'Number of classes in the mixed dataset are: '
Arguments: (24, <class 'int'>)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 284, in joint_train
    logger.info("Number of classes in the mixed dataset are: ", n_classes, type(n_classes))
Message: 'Number of classes in the mixed dataset are: '
Arguments: (24, <class 'int'>)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 287, in joint_train
    logger.info("Seed for random numbers: ", torch.initial_seed())
Message: 'Seed for random numbers: '
Arguments: (1337,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 287, in joint_train
    logger.info("Seed for random numbers: ", torch.initial_seed())
Message: 'Seed for random numbers: '
Arguments: (1337,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 290, in joint_train
    logger.info("Overriding the number of embedding features to: ", n_txt_feats)
Message: 'Overriding the number of embedding features to: '
Arguments: (100,)
--- Logging error ---
Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 980, in emit
    msg = self.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 830, in format
    return fmt.format(record)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 567, in format
    record.message = record.getMessage()
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/logging/__init__.py", line 330, in getMessage
    msg = msg % self.args
TypeError: not all arguments converted during string formatting
Call stack:
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 290, in joint_train
    logger.info("Overriding the number of embedding features to: ", n_txt_feats)
Message: 'Overriding the number of embedding features to: '
Arguments: (100,)
Joint training
Setting the lr to : 0.01
/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py:148: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  yhat = nn.functional.softmax(yhat)
Mixed_ng20---ag_news_0.1 - Iter [0/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.0703125} ; val-metrics- {'logloss': -1, 'accuracy': 0.049642573471008734}
Mixed_ng20---ag_news_0.1 - Iter [1000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.3046875} ; val-metrics- {'logloss': -1, 'accuracy': 0.0}
Mixed_ng20---ag_news_0.1 - Iter [2000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.078125} ; val-metrics- {'logloss': -1, 'accuracy': 0.06155679110405084}
Mixed_ng20---ag_news_0.1 - Iter [3000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.546875} ; val-metrics- {'logloss': -1, 'accuracy': 0.0}
Mixed_ng20---ag_news_0.1 - Iter [4000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.6484375} ; val-metrics- {'logloss': -1, 'accuracy': 0.07744241461477364}
Mixed_ng20---ag_news_0.1 - Iter [5000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.3984375} ; val-metrics- {'logloss': -1, 'accuracy': 0.1699761715647339}
Mixed_ng20---ag_news_0.1 - Iter [6000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.8671875} ; val-metrics- {'logloss': -1, 'accuracy': 0.021445591739475776}
Mixed_ng20---ag_news_0.1 - Iter [7000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.78125} ; val-metrics- {'logloss': -1, 'accuracy': 0.5043685464654488}
Mixed_ng20---ag_news_0.1 - Iter [8000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.9765625} ; val-metrics- {'logloss': -1, 'accuracy': 0.006354249404289118}
Mixed_ng20---ag_news_0.1 - Iter [9000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.890625} ; val-metrics- {'logloss': -1, 'accuracy': 0.07664813343923749}
Mixed_ng20---ag_news_0.1 - Iter [10000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.921875} ; val-metrics- {'logloss': -1, 'accuracy': 0.5035742652899127}
Mixed_ng20---ag_news_0.1 - Iter [11000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.9375} ; val-metrics- {'logloss': -1, 'accuracy': 0.017871326449563147}
Mixed_ng20---ag_news_0.1 - Iter [12000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.9375} ; val-metrics- {'logloss': -1, 'accuracy': 0.6413820492454329}
Mixed_ng20---ag_news_0.1 - Iter [13000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.953125} ; val-metrics- {'logloss': -1, 'accuracy': 0.17116759332803813}
Mixed_ng20---ag_news_0.1 - Iter [14000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.04447974583002383}
Mixed_ng20---ag_news_0.1 - Iter [15000/50000]: train-metrics- {'logloss': -1, 'accuracy': 0.9921875} ; val-metrics- {'logloss': -1, 'accuracy': 0.6969817315329627}
new lr: 0.005
Mixed_ng20---ag_news_0.1 - Iter [16000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.035742652899126294}
Mixed_ng20---ag_news_0.1 - Iter [17000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.20214455917394758}
Mixed_ng20---ag_news_0.1 - Iter [18000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.574265289912629}
Mixed_ng20---ag_news_0.1 - Iter [19000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03216838760921366}
Mixed_ng20---ag_news_0.1 - Iter [20000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7172359015091342}
Mixed_ng20---ag_news_0.1 - Iter [21000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03415409054805401}
Mixed_ng20---ag_news_0.1 - Iter [22000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.05242255758538523}
Mixed_ng20---ag_news_0.1 - Iter [23000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7148530579825259}
Mixed_ng20---ag_news_0.1 - Iter [24000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.029785544082605243}
Mixed_ng20---ag_news_0.1 - Iter [25000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7196187450357426}
Mixed_ng20---ag_news_0.1 - Iter [26000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.28038125496425736}
Mixed_ng20---ag_news_0.1 - Iter [27000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03613979348689436}
Mixed_ng20---ag_news_0.1 - Iter [28000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.721604447974583}
Mixed_ng20---ag_news_0.1 - Iter [29000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03137410643367752}
Mixed_ng20---ag_news_0.1 - Iter [30000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.130659253375695}
new lr: 0.0025
Mixed_ng20---ag_news_0.1 - Iter [31000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.6886417791898332}
Mixed_ng20---ag_news_0.1 - Iter [32000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.021842732327243843}
Mixed_ng20---ag_news_0.1 - Iter [33000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7208101667990469}
Mixed_ng20---ag_news_0.1 - Iter [34000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.15051628276409848}
Mixed_ng20---ag_news_0.1 - Iter [35000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.04169976171564734}
Mixed_ng20---ag_news_0.1 - Iter [36000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7188244638602065}
Mixed_ng20---ag_news_0.1 - Iter [37000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.026211278792692614}
Mixed_ng20---ag_news_0.1 - Iter [38000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.28832406671961874}
Mixed_ng20---ag_news_0.1 - Iter [39000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.4654487688641779}
Mixed_ng20---ag_news_0.1 - Iter [40000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03216838760921366}
Mixed_ng20---ag_news_0.1 - Iter [41000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7235901509134234}
Mixed_ng20---ag_news_0.1 - Iter [42000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03137410643367752}
Mixed_ng20---ag_news_0.1 - Iter [43000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.06473391580619539}
Mixed_ng20---ag_news_0.1 - Iter [44000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7128673550436855}
Mixed_ng20---ag_news_0.1 - Iter [45000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.025416997617156472}
new lr: 0.00125
Mixed_ng20---ag_news_0.1 - Iter [46000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.721604447974583}
Mixed_ng20---ag_news_0.1 - Iter [47000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.21445591739475775}
Mixed_ng20---ag_news_0.1 - Iter [48000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.03494837172359015}
Mixed_ng20---ag_news_0.1 - Iter [49000/50000]: train-metrics- {'logloss': -1, 'accuracy': 1.0} ; val-metrics- {'logloss': -1, 'accuracy': 0.7212073073868149}
After Training: Testing on root dataset only
Traceback (most recent call last):
  File "/home/usaxena/.local/lib/python3.5/site-packages/torch/nn/modules/module.py", line 482, in load_state_dict
    own_state[name].copy_(param)
RuntimeError: invalid argument 2: sizes do not match at /pytorch/torch/lib/THC/generic/THCTensorCopy.c:51

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 184, in _run_module_as_main
    "__main__", mod_spec)
  File "/cm/shared/apps/python/3.5.2/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/main.py", line 87, in <module>
    joint_train(opt, logger)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 314, in joint_train
    test(model, logger, opt, root_te_data, n_classes, dataset_name)
  File "/home/usaxena/work/s18/lex/code/vdcnn/src/VDCNN.py", line 261, in test
    model.load_state_dict(checkpoint['model'])
  File "/home/usaxena/.local/lib/python3.5/site-packages/torch/nn/modules/module.py", line 487, in load_state_dict
    .format(name, own_state[name].size(), param.size()))
RuntimeError: While copying the parameter named embed.weight, whose dimensions in the model are torch.Size([100, 16]) and whose dimensions in the checkpoint are torch.Size([66, 16]).
